### DISCUSSION TITLE
change "Device" name for multiple "inventory item"

### DISCUSSION THREAD

**Discussion by ghorban-mohajeri** [netbox-community/netbox#7526] [Q&A]
Hello dear
I want to edit multiple inventory items  together , I select multiple "inventory items" and select "edit selected" key , 
But I cant edit "Device" name for them.
Is it any way to do that , I can change the "Device" name for single "inventory items" , but I want to change it for multiple "inventory items" .

Thanks in advanced
![inventory](https://user-images.githubusercontent.com/76703528/136920181-c8dcc2e6-93d3-4b39-95df-a4beeb4d7ddf.jpg)

**Comment by candlerb** [comment]
You want to move multiple inventory items from one device to another, is that correct?  (That is, you're not editing the device name - you're changing the link to the device they're associated with).

It seems like a reasonable feature request to me.

**Reply by ghorban-mohajeri** [reply]
yes , exactly , I want to move multiple inventory items from one device to another , all together.
How can I do that (with ansible or other way) ?

**Reply by candlerb** [reply]
For now, you could do it using the REST API, or a custom script (using the Django ORM), or a standalone script (using nbshell and the Django ORM).

**Reply by ghorban-mohajeri** [reply]
What about feature ? 
When do you think this featurs will add to the netbox ?

**Reply by candlerb** [reply]
First step is to raise it as a feature request on github.
